appcircle_ios_reactnative_cache_push/1.0.2/component.yaml

```diff
@@ -0,0 +1,39 @@
+platform: iOS
+buildPlatform: ReactNative
+displayName: "Cache Push"
+description: "Uploads user selected files and folders to Appcircle cache."
+docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps#cache-push
+requiredComponents: "appcircle_cocoapods,appcircle_npm_yarn,appcircle_node_install"
```

Comment:
@boztopuz I think we can move these in `precedingComponents`list because these dependencies may vary depending on the input entered.